knowledge_graph/routes.py

```diff
@@ -72,9 +58,14 @@ def query() -> Tuple[Response, int]:
 
 
 @app.route("/questions", methods=["GET"])
+@auto.doc()
 def get_questions() -> Tuple[Response, int]:
+    """
+    Returns the list of available questions that can be presented to
+    the user.
+    """
     try:
-        with open("schwartz_questions.json") as json_file:
+        with open("json_files/schwartz_questions.json") as json_file:
```

Comment:
For computer file paths they can’t be coded with “/“ because it won’t work on windows file systems (will break for devs doing localhost on windows).

Must replace everywhere in the code any file paths with “/“ (this doesn’t apply for urls though!).
Replace each one using the format:

    path_corrected = os.path.join(directory, child_folder, file_name) 

Don’t forget to also import os at the top of the file too